src/test/java/com/example/HelloWorldResourceTest.java

```diff
@@ -19,41 +25,71 @@ public class HelloWorldResourceTest {
             ResourceHelpers.resourceFilePath("config/test-it.yml")
     );
 
-//    private static WireMockServer wireMockServer;
-
-//    @BeforeClass
-//    public static void setUp() {
-//        // Start WireMock server on port 8080
-//        wireMockServer = new WireMockServer(8080);
-//        wireMockServer.start();
-//    }
-
     @Test
     void ShouldReturnHelloWorldText() {
         WireMockServer wireMockServer = new WireMockServer(Integer.parseInt(extension.getConfiguration().getNodePort()));
         wireMockServer.start();
-        // Stub the API response using WireMock
+
+
         configureFor("localhost", Integer.parseInt(extension.getConfiguration().getNodePort()));
         stubFor(get((urlEqualTo("/")))
                 .willReturn(aResponse()
                         .withStatus(200)
                         .withHeader("Content-Type", "application/json")
-                        .withBody("{\"text\": \"Hello, world!\"}")));
+                        .withBody("""
+                                {
+                                  "text": "Hello, world!"
+                                }
+                                """)));
 
-        // Send a request to the API using REST Assured
-            given()
+
+        given()
                 .port(extension.getLocalPort())
                 .when()
                 .get("/")
                 .then()
                 .statusCode(200)
                 .contentType("application/json")
                 .body("text", is("Hello, world!"));
+
         wireMockServer.stop();
     }
 
-//    @AfterClass
-//    public static void setDown() {
-//        wireMockServer.stop();
-//    }
+
+    @Test
+    void shouldReturn200AndGreetingSentToPostGreetingNodeEndpoint() {
+
+        WireMockServer wireMockServer = new WireMockServer(Integer.parseInt(extension.getConfiguration().getNodePort()));
+        wireMockServer.start();
```

Comment:
So here you're starting up and shutting down a WireMockServer individually in each class. Is there a way you can extract this out to a `@BeforeAll` and `@AfterAll`?